lib/send_reset_email.rb

```diff
@@ -0,0 +1,24 @@
+require 'mailgun'
+
+class SendResetEmail
+
+  def initialize(user)
+    @user = user
+    @client = Mailgun::Client.new(ENV['MAILGUN_API_KEY'])
+  end
+
+  def send_message
+    RestClient::Request.execute(
+      :url => "https://api:#{ENV['MAILGUN_API_KEY']}@api.mailgun.net/v3/sandbox119f7055446a45519d466456e9412df8.mailgun.org/messages",
```

Comment:
Metrics/LineLength: Line is too long. [134/90]<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax.